stubs/Pygments/pygments/sphinxext.pyi

```diff
@@ -8,11 +8,11 @@ FMTERDOC: str
 FILTERDOC: str
 
 class PygmentsDoc(Directive):
-    has_content: bool
-    required_arguments: int
-    optional_arguments: int
-    final_argument_whitespace: bool
-    option_spec: Any
+    has_content: ClassVar[bool]
+    required_arguments: ClassVar[int]
+    optional_arguments: ClassVar[int]
+    final_argument_whitespace: ClassVar[bool]
+    option_spec: ClassVar[Any]
```

Comment:
```suggestion
```

these are all defined on the `Directive` base class so can be removed.

Although `Directive` will need to be modified to set the members to `ClassVar`s

Reply:
are there any other subclasses that should have the redefinitions removed from their type stubs?